extensions/schema-compare/package.json

```diff
@@ -54,6 +54,11 @@
           "command": "schemaCompare.start",
           "when": "connectionProvider == MSSQL && nodeType && nodeType == Database && mssql:engineedition != 11",
           "group": "export"
+        },
+        {
+          "command": "schemaCompare.start",
+          "when": "connectionProvider == MSSQL && nodeType && nodeType == Server && databaseName != 'master' && databaseName != '' && mssql:engineedition != 11",
```

Comment:
How much do you care about this being active for other system DBs such as model? We could consider adding a isSystemDb and isDefaultDb keys to help make these kind of checks easier and more robust.

Reply:
We didn't care that much when first making the extensions, but SSMS doesn't show DacFx actions in the context menu for system dbs so it would be good to align with that.

Reply:
after talking with Alan, we decided we don't really care about if it's a system db and we should allow customers to use schema compare and the dacfx wizard on them if they want to